src/snippy.ts

```diff
@@ -110,7 +120,7 @@ export class Snippy {
     const snippet = {
       name: fileName,
       lang: LANG_MAP_DEFAULTS[fileName.split('.').pop()],
-      content: contentArr.join('\n'),
+      content: contentArr.join('\n').replace(/\n\n\n/g, '\n\n'),
```

Comment:
since this replace looks a little unusual, it would be good to provide a comment explaining what/why. Or move the behaviour to a function so the intention is clear.

Reply:
I forgot to remove this, fixed.